pkg/kubelet/kubelet.go

```diff
@@ -1171,7 +1171,9 @@ func (kl *Kubelet) syncPod(pod *api.Pod, mirrorPod *api.Pod, runningPod kubecont
 	var podStatus api.PodStatus
 	if updateType == SyncPodCreate {
 		podStatus = pod.Status
-		glog.V(3).Infof("Not generating pod status for new pod %v", podFullName)
+		podStatus.StartTime = &util.Time{start}
```

Comment:
I think setPodStatus will already set the time if it's unset. 

The impact here is extra qps, because we'll send one update for the time stamp and one update for the running in the defer call, but I'm hoping it won't be too bad. 

So I actually wanted to do something a little different (just mentioning it here, not sure if you want to do it): Call SetPodStatus here with the actual pod.Status given to this method, and set the StartTime on that pod when we receive the watch notification (https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubelet/config/config.go#L264 is close enough to the entry point). That would help our e2e tests gather latency numbers, but I'm not going to push too hard for it because it might shortchange the activeDeadline feature :)